OpenRA.Game/Network/Order.cs

```diff
@@ -167,6 +167,11 @@ public static Order CancelProduction(Actor subject, string item, int count)
 			return new Order("CancelProduction", subject, false) { ExtraData = (uint)count, TargetString = item };
 		}
 
+		public static Order NewAdmin(string admin)
+		{
+			return new Order("NewAdmin", null, false) { IsImmediate = false, TargetString = admin };
```

Comment:
Two easy to fix issues here:
* IsImmediate defaults to false, so doesn't need to be specified
* The admin client name isn't robust (it might change in the lobby, and is not unique), so please use the client ID instead.

Reply:
Fixed.